env/env.js

```diff
@@ -38,6 +38,7 @@ const base = removeNullValues({
   REACT_APP_FEEDBACK_ADDR: 'feedback@votebymail.io',
   REACT_APP_ELECTION_OFFICIAL_ADDR: 'elections@votebymail.io',
   REACT_APP_SUPPORT_ADDR: 'support@votebymail.io',
+  REACT_APP_RECORDS_ADDR: 'records@votebymail.io',
```

Comment:
I wouldn't put this in base.  You really only want emails from production right?  So I would only put it in production.

Also, REACT_APP_RECORDS_ADDR should not have REACT_APP prefix since we don't use it in the front end.